components/player/VideoPlayer.tsx

```diff
@@ -1,15 +1,65 @@
-import React from 'react';
-import { View } from 'react-native';
+/* eslint-disable react/no-unstable-nested-components */
+import { useTheme } from '@react-navigation/native';
+import { ResizeMode, Video } from 'expo-av';
+import { Image } from 'expo-image';
+import React, { useRef } from 'react';
+import { Dimensions, StyleSheet, View } from 'react-native';
+import { VIDEO_URL } from '../../config/constants';
 
-// TODO: Add VideoPlayer component
-/**
- * -  Implement a video player card component
- * - Hint: use `expo-av` to implement the video player with hidden controls
- * - consider adding fullscreen player
- */
+const { width } = Dimensions.get('window');
 
 const VideoPlayer = () => {
-  return <View />;
+  const { colors } = useTheme();
+
+  const video = useRef(null);
+
+  return (
+    <View style={[styles.container, { backgroundColor: colors.card }]} testID="video-player">
+      <Video
+        testID="video-component"
+        ref={video}
+        style={styles.video}
+        source={{
+          uri: VIDEO_URL,
+        }}
+        useNativeControls
+        resizeMode={ResizeMode.COVER}
```

Comment:
We need to use resizeMode `CONTAIN` when we switch to full screen and `COVER` when we switch back to normal. On android, the video fills up the entire screen and is cut off currently on full screen. Alternatively, what I figure out from the design is that if we have the custom mute/unmute button, we have to hide the native controls anyways to avoid overlapping icons on iOS, in which case this is no longer a problem.

![telegram-cloud-photo-size-4-5834776105012280526-y](https://github.com/droidconKE/droidconKE2023ReactNative/assets/5578688/315f5674-eb00-4d80-8b36-55de27ec6559)